src/main/java/amazin/controller/BookController.java

```diff
@@ -97,48 +97,23 @@ public String deleteBook(@PathVariable("id") Long id, RedirectAttributes attribu
         return createRedirectViewPath(REQUEST_MAPPING_BOOK);
     }
 
-    @GetMapping("/fuzzySearchName")
-    public String fuzzySearchByBookName(@RequestParam(value = "keywords", required = false) String text, Model model) {
-        List<Book> searchResults = null;
-        try {
-            searchResults = searchservice.fuzzySearch(text, "name");
-
-        } catch (Exception ex) {
-            // for now do nothing. Later we will throw an exception and display a message to
-            // the user
-        }
-        model.addAttribute(MODEL_ATTRIBUTE_BOOK, searchResults);
-        return createRedirectViewPath(REQUEST_MAPPING_BOOK);
-    }
-
-    @GetMapping("/keywordSearchName")
-    public String keywordSearchByBookName(@RequestParam(value = "keywords", required = false) String text,
-            Model model) {
-        List<Book> searchResults = null;
-        try {
-            searchResults = searchservice.keywordSearch(text, "name");
+    @GetMapping("/searchByName")
```

Comment:
can we update BookControllerTest for this new method addition?

Reply:
I couldn't find any documentation to do this properly so this won't be done, at least for now.